### PR TITLE
update antora structure

### PR DESCRIPTION
**Pull request by laurafitzgerald**
### Motivation
Update the antora structure to match other SDK docs format/syntax

### Progress

- [x] Move docs under /docs/modules/ROOT/pages
- [x] Remove all other folders under modules
- [x] update README to use new routes
- [x] small adoc fixes

### CODE REVIEW

**Comment by wtrocki** [comment]
@laurafitzgerald Really excited to see you contributing to IOS SDK! Great job!

**Comment by wtrocki** [comment]
> @finp there are some docs still under docs/contrib I wasn't sure how to approach those. Were there similar files in the android one?

Do not touch them. This what we agreed on the list with @finp. It's literally safe heaven for developers who can put any content without exposing them to end users. We document there release process etc. More info on the mailing list.

Verified with: `antora local-site.yml`

**Comment by laurafitzgerald** [comment]
@wtrocki thanks for the reivew. I did move the `core.adoc` documentation from the `contrib` folder. I think the `extending-metrics.adoc` file is a candidate to be moved as well... wdyt? I did some other small fixes if could have some time to look at those changes. thanks :)

**Comment by wtrocki** [comment]
@laurafitzgerald Please do not move them! There are internal as we mentioned on the mailing list. Let's revert this changes. `extending-metrics.adoc ` should not be moved. We had trello card to hide metrics from users etc.

File: README.md [hunk 1]
```diff
@@ -15,13 +15,13 @@ To try out latest release of SDK go to [IOS showcase application](https://github
 
 AeroGear Services consist of set of separate SDKs
 
-- [Core](./docs/core/README.adoc)
+- [Core](./docs/modules/ROOT/pages/core.adoc)
```
Comment (wtrocki) [inline README.md]: This actually may be deleted now. We do not need to expose.

Comment (laurafitzgerald) [reply README.md]: @wtrocki thanks for the info. Is there a different name that we can use for this one as it follows the syntax of the other ones. e.g. core, auth, sync....

**Comment by laurafitzgerald** [comment]
@wtrocki will move those back in a new commit, thanks for info

File: README.md (hunk 1 above)
Comment (wtrocki) [reply README.md]: Not sure what you mean.

Comment (laurafitzgerald) [reply README.md]: I made the assumption that core.adoc was instructions on how to use the Core SDK as the auth.adoc has instructions on how to use the Auth SDK. Chatting with @finp and he has clarified that one for me. He also has some suggestions in terms of naming conventions for docs across the SDKs which he is sending to aerogear mailing list. I'll wait for response to that to make further changes here.

**Comment by laurafitzgerald** [comment]
Linking: https://github.com/aerogear/aerogear-ios-sdk/pull/117 as changes are related.

**Comment by wtrocki** [comment]
@laurafitzgerald World class job! Leaving @finp to do final approval!

**Pull request merged**